x-pack/plugins/ml/public/application/supplied_configurations/supplied_configurations_flyout/overview_tab_content.tsx

```diff
@@ -0,0 +1,352 @@
+                  }
+                />
+              )}
+
+              <EuiSpacer size="m" />
+              <EuiDescriptionListTitle>
+                <FormattedMessage
+                  id="xpack.ml.anomalyDetection.suppliedConfigurationsFlyout.typeTitle"
+                  defaultMessage="Type"
+                />
+                <EuiSpacer size="s" />
+              </EuiDescriptionListTitle>
+              <EuiDescriptionListDescription>{module.type}</EuiDescriptionListDescription>
+            </EuiDescriptionList>
+          </EuiFlexItem>
+
+          {/* COLUMN WITH QUERY */}
+          <EuiFlexItem grow={3}>
+            {module.query !== undefined ? (
+              <EuiFlexGroup direction="column" gutterSize="xl">
+                <EuiFlexItem>
+                  <EuiDescriptionList rowGutterSize="m">
+                    <EuiDescriptionListTitle>
+                      <FormattedMessage
+                        id="xpack.ml.anomalyDetection.suppliedConfigurationsFlyout.eligibleTitle"
+                        defaultMessage="One or more indices must match the following query:"
+                      />
+                    </EuiDescriptionListTitle>
+                    <EuiSpacer size="m" />
+                    <EuiDescriptionListDescription>
+                      <EuiCodeBlock
+                        language="json"
+                        isCopyable
+                        overflowHeight="500px"
+                        data-test-subj="mlPreconfigJobsQueryBlock"
+                      >
+                        {JSON.stringify(module.query, null, 2)}
+                      </EuiCodeBlock>
+                      <EuiSpacer size="s" />
+                    </EuiDescriptionListDescription>
+                  </EuiDescriptionList>
+                </EuiFlexItem>
+                {module.query !== undefined ? (
+                  <EuiFlexItem grow={false}>
+                    <EuiFlexGroup gutterSize="m" alignItems="center" justifyContent="flexEnd">
+                      <EuiFlexItem grow={false}>
+                        <EuiText size={'xs'}>
+                          <FormattedMessage
+                            id="xpack.ml.anomalyDetection.suppliedConfigurationsFlyout.dataRecognizerHelpButtonLabel"
+                            defaultMessage="Check indices to see which can be used to run jobs in this module?"
+                          />
+                        </EuiText>
+                      </EuiFlexItem>
+                      <EuiFlexItem grow={false}>
+                        <span>
+                          <EuiButton
+                            fill
+                            isDisabled={runningDataRecognizer}
+                            isLoading={runningDataRecognizer}
+                            color={'primary'}
+                            onClick={runDataRecongizer}
+                            size="s"
+                          >
+                            <FormattedMessage
```

Comment:
Have you tested this against the Kibana sample data web logs module? I have add the sample web logs data but the recognizer is returning no matching data views in the flyout, whereas it works as expected inside the job wizards.

Reply:
yeah - should be working 🤔 
![image](https://github.com/user-attachments/assets/2803bda1-ac84-4d22-a40c-cdc365f18e58)

Reply:
@peteharverson - this is happening because the endpoint doesn't check against indices starting with `.` since those are internal indices. A user cannot name an index starting with `.`. Looks like this sample index is named starting with an `.`.  I think if this is just an edge case then we should keep it as is. I will confirm if there's a way a user could potentially name an index beginning with `.`.

Reply:
You're right - the Kibana sample data logs use the `kibana_sample_data_logs` data stream, which is backed by the `.ds-kibana_sample_data_logs-` style index. I don't think we want to change the behavior here to check against `.` indices.

I guess we'd have to change the check to run against data views (and then against the corresponding index pattern _or_ data stream backed index), but that is probably outside the scope of this PR.